pyqtorch/core/utils.py

```diff
@@ -12,15 +12,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import torch
-import numpy as np
 from string import ascii_letters as ABC
+from typing import Any, Sized
 
-ABC_ARRAY = np.array(list(ABC))
+import numpy as np
+import torch
+from numpy.typing import NDArray
 
+ABC_ARRAY: NDArray = np.array(list(ABC))
```

Comment:
@awennersteen I do not really get why we need such an array. Why can't we just use a standard Numpy array in its place?

Reply:
My only work here was only adding the typing, this was Slimane. 
I'm not sure what's the need but we can probably find out from Slimane.

Reply:
@awennersteen Got it. Let's keep it as it is for now then.